lib/internal/test_runner/runner.js

```diff
@@ -112,47 +121,131 @@ function getRunArgs({ path, inspectPort }) {
   return argv;
 }
 
+class FileTest extends Test {
+  #buffer = [];
+  #handleReportItem({ kind, node, level = 0, lexeme, tokens }) {
+
+    const indent = StringPrototypeRepeat(kDefaultIndent, level + 1);
+
+    const details = (diagnostic) => {
+      return (
+        diagnostic && {
+          __proto__: null,
+          yaml:
+            `${indent}  ` +
+            ArrayPrototypeJoin(diagnostic, `\n${indent}  `) +
+            '\n',
+        }
+      );
+    };
+
+    switch (kind) {
+      case TokenKind.TAP_VERSION:
+        break; // skip
+
+      case TokenKind.TAP_PLAN:
+        this.reporter.plan(indent, node.plan.end - node.plan.start + 1);
+        break;
+
+      case TokenKind.TAP_SUBTEST_POINT:
+        this.reporter.subtest(indent, node.name);
+        break;
+
+      case TokenKind.TAP_TEST_OK:
+      case TokenKind.TAP_TEST_NOTOK:
+        const { todo, skip, pass, fail } = node.test.status;
+        const testName = node.test.description;
+        const testnumber = node.test.id;
+        let directive = {};
+
+        if (todo) {
+          this.skip(node.test.reason);
```

Comment:
why `this.skip`?

Reply:
Removed. See https://github.com/nodejs/node/pull/43525/files/59ccdfeb367a7f81cb1a5f452d10082498dd993b#r978371320